scripts/quickstart.sh

```diff
@@ -46,6 +46,12 @@ quickstart() {
     version=$(cat version.txt | tr -d '\n')
     echo -e "${green}ark${nc} quickstart ${white}v${version}${nc}"
 
+    if [ "${FORCE:-false}" != true ]; then
```

Comment:
phew finally, this is great. maybe we also note that it for local development use `devspace dev`